demo/node/src/chain_spec.rs

```diff
@@ -56,6 +56,7 @@ pub fn pc_create_chain_spec(config: &CreateChainSpecConfig<SessionKeys>) -> serd
 			participation_data_release_period: 30,
 			..Default::default()
 		},
+		bridge: Default::default(),
```

Comment:
I've added a JIRA ticket to update wizards, chain-spec CI jobs and local-environment